filters/ThirdParty/filter_122_FanboysAnnoyances/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "filterId": 122,
   "name": "Fanboy's Annoyances",
-  "description": "removes online annoyances (includes Fanboy's Social Blocking List) — supplement for EasyList",
+  "description": "Removes in-page pop-ups and other annoyances. (Includes Fanboy's Social Blocking & Cookiemonster Lists)",
```

Comment:
I guess there in no need in parentheses in such case

Reply:
this is additional info

Reply:
choose one:
```
Removes in-page pop-ups and other annoyances. Includes Fanboy's Social Blocking & Cookiemonster Lists.
```

```
Removes in-page pop-ups and other annoyances (includes Fanboy's Social Blocking & Cookiemonster Lists).
```

Reply:
done. Changed 3 descriptions